dist/map-panel.js

```diff
@@ -181,24 +179,6 @@ Fliplet.InteractiveMap.component('map-panel', {
   }
 });
 
-Fliplet.Widget.onCancelRequest(function () {
```

Comment:
why are these gone away?

Reply:
@MaksymShokin please link relevant follow up PR to restore these lines

Reply:
@squallstar PR that restores deleted lines https://github.com/Fliplet/fliplet-widget-interactive-map/pull/60.